server/core/region.go

```diff
@@ -23,11 +23,17 @@ import (
 	"unsafe"
 
 	"github.com/gogo/protobuf/proto"
+	"github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/kvproto/pkg/pdpb"
 	"github.com/pingcap/kvproto/pkg/replication_modepb"
 )
 
+// errRegionIsStale is error info for region is stale.
+var errRegionIsStale = func(region *metapb.Region, origin *metapb.Region) error {
+	return errors.Errorf("region is stale: region %v origin %v", region, origin)
```

Comment:
why do you use `Errorf`?

Reply:
What do you mean?

Reply:
I just keep the original implementation in errors.go

Reply:
seems it is useless.

Reply:
We will use it in `PreCheckPutRegion`